packages/installations/src/api/create-installation.test.ts

```diff
@@ -104,24 +105,41 @@ describe('createInstallation', () => {
   });
 
   describe('failed request', () => {
-    beforeEach(() => {
-      const response = {
+    it('throws a FirebaseError with the error information from the server', async () => {
+      const errorResponse: ErrorResponse = {
         error: {
           code: 409,
           message: 'Requested entity already exists',
           status: 'ALREADY_EXISTS'
         }
       };
 
-      fetchSpy = stub(self, 'fetch').resolves(
-        new Response(JSON.stringify(response), { status: 409 })
+      fetchSpy.resolves(
+        new Response(JSON.stringify(errorResponse), { status: 409 })
       );
-    });
 
-    it('throws a FirebaseError with the error information from the server', async () => {
       await expect(
         createInstallation(appConfig, inProgressInstallationEntry)
       ).to.be.rejectedWith(FirebaseError);
     });
+
+    it('retries once if the server returns a 5xx error', async () => {
```

Comment:
as well as a test that it does retry once, we want tests that it doesn't retry infinitely?

Reply:
Added tests for `retryIfServerError`. No need to duplicate them in all API functions.